### PR TITLE
New version 3.1.1 released

### PR DESCRIPTION
**Pull request by wisecleaner**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/69304)

### CODE REVIEW

**Comment by wisecleaner** [comment]
Hi, Why did this happen? "The manifest version is not supported." I used the YAMLCreate.ps1 powershell script located in the Tools folder to create files, and ran the command "winget validate" to verify.

**Comment by SpecterShell** [comment]
https://github.com/microsoft/winget-pkgs/issues/68845

**Review by ItzLevvie — DISMISSED** [review]

**Comment by wisecleaner** [comment]
@SpecterShell Thanks.
What should I do now? 
Waiting for the Manifest schema 1.2 portable packages start being accepted?
Or, switch the Manifest Version back to 1.1.0 in the PR? How to do it? Please advice.
I'm the first time to submit an app to Winget.
Thank you.

File: manifests/w/WiseCleaner/WiseProgramUninstaller/3.1.1/WiseCleaner.WiseProgramUninstaller.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+InstallerSwitches:
+  Silent: /verysilent
+UpgradeBehavior: install
+ReleaseDate: 2022-08-10
+Installers:
+- Architecture: x86
+  InstallerUrl: https://downloads.wisecleaner.com/soft/WPUSetup_3.1.1.253.exe
+  InstallerSha256: 5D35496C2D085F9FFEC34BDF7DC21264BCF8B5EEABAF748F7C805EE912FBC525
+ManifestType: installer
+ManifestVersion: 1.2.0
```
Comment (SpecterShell) [inline manifests/w/WiseCleaner/WiseProgramUninstaller/3.1.1/WiseCleaner.WiseProgramUninstaller.installer.yaml:22]:
```suggestion
ManifestVersion: 1.1.0
```

File: manifests/w/WiseCleaner/WiseProgramUninstaller/3.1.1/WiseCleaner.WiseProgramUninstaller.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+- uninstaaller
+- wpu
+# Agreements: 
+ReleaseNotes: https://www.wisecleaner.com/blog_content_1101.html
+ReleaseNotesUrl: https://www.wisecleaner.com/blog_content_1101.html
+# PurchaseUrl: 
+# InstallationNotes: 
+# Documentations: 
+ManifestType: defaultLocale
+ManifestVersion: 1.2.0
```
Comment (SpecterShell) [inline manifests/w/WiseCleaner/WiseProgramUninstaller/3.1.1/WiseCleaner.WiseProgramUninstaller.locale.en-US.yaml:32]:
```suggestion
ManifestVersion: 1.1.0
```

File: manifests/w/WiseCleaner/WiseProgramUninstaller/3.1.1/WiseCleaner.WiseProgramUninstaller.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.1.3 $debug=NVS1.5-1-22621-436
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: WiseCleaner.WiseProgramUninstaller
+PackageVersion: 3.1.1
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.2.0
```
Comment (SpecterShell) [inline manifests/w/WiseCleaner/WiseProgramUninstaller/3.1.1/WiseCleaner.WiseProgramUninstaller.yaml:8]:
```suggestion
ManifestVersion: 1.1.0
```

File: manifests/w/WiseCleaner/WiseProgramUninstaller/3.1.1/WiseCleaner.WiseProgramUninstaller.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# Created with YamlCreate.ps1 v2.1.3 $debug=NVS1.5-1-22621-436
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
```
Comment (SpecterShell) [inline manifests/w/WiseCleaner/WiseProgramUninstaller/3.1.1/WiseCleaner.WiseProgramUninstaller.installer.yaml:2]:
```suggestion
# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
```

File: manifests/w/WiseCleaner/WiseProgramUninstaller/3.1.1/WiseCleaner.WiseProgramUninstaller.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,32 @@
+# Created with YamlCreate.ps1 v2.1.3 $debug=NVS1.5-1-22621-436
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
```
Comment (SpecterShell) [inline manifests/w/WiseCleaner/WiseProgramUninstaller/3.1.1/WiseCleaner.WiseProgramUninstaller.locale.en-US.yaml:2]:
```suggestion
# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
```

File: manifests/w/WiseCleaner/WiseProgramUninstaller/3.1.1/WiseCleaner.WiseProgramUninstaller.yaml [hunk 6]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.1.3 $debug=NVS1.5-1-22621-436
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
```
Comment (SpecterShell) [inline manifests/w/WiseCleaner/WiseProgramUninstaller/3.1.1/WiseCleaner.WiseProgramUninstaller.yaml:2]:
```suggestion
# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
```

**Comment by wisecleaner** [comment]
Change all "1.2.0" to "1.1.0" in 3 files？
Okay, I will try it. Thank you very much!

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**